airflow/config_templates/default_airflow.cfg

```diff
@@ -658,6 +658,18 @@ auto_refresh_interval = 3
 # Boolean for displaying warning for publicly viewable deployment
 warn_deployment_exposure = True
 
+# Number of dags to look back for dag audit log view
+audit_trail_query_days = 31
```

Comment:
Is the trail necessary if we have to limit the number anyway? I don't think we use "days back" anywhere else in the UI. It would be best not to introduce new paradigms.

Reply:
Agreed - if people have large volumes of logs then they may introduce their own log rotation / table cleanup to deal with this, but I don't think that this needs to be handled here. 

It also introduces an artificial limitation on the audit view, as users will be unable to view old events even if they still existent in the database. If you're really worried about performance or table size here maybe we could implement query-level pagination?